src/smartpeak/include/SmartPeak/core/Utilities.h

```diff
@@ -287,5 +287,19 @@ namespace SmartPeak
       @returns The numbers of elements found
     */
     static size_t directorySize(const std::string& pathname);
+
+    /**
+    * @brief Constructs an absolute filepath to application logs.
+    * 
+    * Default locations of logs:
+    *   - Windows: C:\Users\<user>\AppData\Local\SmartPeak
+    *   - Linux and MacOS: ~/.SmartPeak
+    * User can change default location and specify directory where the logs are stored by
+    * setting SMARTPEAK_LOGS env variable.
```

Comment:
I think @bertrandboudaud already mentioned this but can you make a note of how to change the log location via the environmental variable in the README?